x-pack/docs/en/security/auditing/event-types.asciidoc

```diff
@@ -43,189 +45,458 @@ multiple audit events. The following is a list of the events that can be generat
 [[audit-event-attributes]]
 === Audit event attributes
 
-The following table shows the common attributes that can be associated with every event.
+In 6.5.0, there is a new <<audit-log-output, `logfile` audit output>> format.
+This format also brings in a few changes for audit event attributes.
+
+The new format is output to the `<clustername>_audit.log` file.
+The audit entries are formatted as flat JSON documents (that is to say, no
+nested objects), one per line. Hence, the attribute names are JSON keys and they
+follow a dotted name syntax. Any attributes that lack a value (`null`) are not
+output.
+
+The following list shows attributes that are common to all audit events.
+Their names and values are analogous to those in the deprecated `logfile` or
+`index` output formats. However, it is expected that the formats will evolve
```

Comment:
Is it correct that the `index` output format is deprecated?  If not, maybe change this phrase to "... the index or deprecated logfile output format..."